README.md

```
@@ -174,6 +238,8 @@ For more details, see [tests/README.md](tests/README.md).
 - [Swagger API Documentation](https://careapi.ohc.network/swagger/)
 - [Testing Documentation](https://docs.coronasafe.network/care-testing-documentation/)
 
+#### Setting up
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)